File: Actividad 1/Clase/Animal.py
class animal:
    def __init__(self,Nombre, Peso, Propietario, fecha_cumpleaños, fecha_ultima_vacuna):
        self._Nombre = Nombre
        self._Peso = Peso
        self._Propietario = Propietario
        self._fecha_cumpleaños = fecha_cumpleaños
        self._fecha_ultima_vacuna = fecha_ultima_vacuna

    #Metodo GET
    @property
    def Nombre(self):
        return self._Nombre
    #Metodo SET
    @Nombre.setter
    def Nombre(self,nuevo_Nombre):
        self._Nombre = nuevo_Nombre
    
    #Metodo GET
    @property
    def Peso(self):
        return self._Peso
    #Metodo SET
    @Peso.setter
    def Peso(self,nuevo_Peso):
        self._Peso = nuevo_Peso
    
    #Metodo GET
    @property
    def Propietario(self):
        return self._Propietario
    #Metodo SET
    @Propietario.setter
    def Propietario(self,nuevo_Propietario):
        self._Propietario = nuevo_Propietario
    
    #Metodo GET
    @property
    def fecha_cumpleaños(self):
        return self._fecha_cumpleaños
    #Metodo SET
    @fecha_cumpleaños.setter
    def fecha_cumpleaños(self,nuevo_fecha_cumpleaños):
        self._fecha_cumpleaños = nuevo_fecha_cumpleaños
    
    #Metodo GET
    @property
    def fecha_ultima_vacuna(self):
        return self._fecha_ultima_vacuna
    #Metodo SET
    @fecha_ultima_vacuna.setter
    def fecha_ultima_vacuna(self,nuevo_fecha_ultima_vacuna):
        self._fecha_ultima_vacuna = nuevo_fecha_ultima_vacuna
    
    


        

File: ACTIVIDAD1-JAVA/src/Clases/Animal.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package Clases;

import java.time.LocalDate;

/**
 *
 * @author Didier
 */
public class Animal {
     private String nombre;
    private double peso;
    private String propietario;
    private LocalDate fechaCumpleaños;
    private LocalDate fechaUltimaVacuna;
    
     public Animal(String nombre, double peso, String propietario, LocalDate fechaCumpleaños, LocalDate fechaUltimaVacuna) {
        this.nombre = nombre;
        this.peso = peso;
        this.propietario = propietario;
        this.fechaCumpleaños = fechaCumpleaños;
        this.fechaUltimaVacuna = fechaUltimaVacuna;
    }

    /**
     * @return the nombre
     */
    public String getNombre() {
        return nombre;
    }

    /**
     * @param nombre the nombre to set
     */
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    /**
     * @return the peso
     */
    public double getPeso() {
        return peso;
    }

    /**
     * @param peso the peso to set
     */
    public void setPeso(double peso) {
        this.peso = peso;
    }

    /**
     * @return the propietario
     */
    public String getPropietario() {
        return propietario;
    }

    /**
     * @param propietario the propietario to set
     */
    public void setPropietario(String propietario) {
        this.propietario = propietario;
    }

    /**
     * @return the fechaCumpleaños
     */
    public LocalDate getFechaCumpleaños() {
        return fechaCumpleaños;
    }

    /**
     * @param fechaCumpleaños the fechaCumpleaños to set
     */
    public void setFechaCumpleaños(LocalDate fechaCumpleaños) {
        this.fechaCumpleaños = fechaCumpleaños;
    }

    /**
     * @return the fechaUltimaVacuna
     */
    public LocalDate getFechaUltimaVacuna() {
        return fechaUltimaVacuna;
    }

    /**
     * @param fechaUltimaVacuna the fechaUltimaVacuna to set
     */
    public void setFechaUltimaVacuna(LocalDate fechaUltimaVacuna) {
        this.fechaUltimaVacuna = fechaUltimaVacuna;
    }
    public void mostrarInfo() {
        System.out.println("Nombre: " + nombre);
        System.out.println("Peso: " + peso + " kg");
        System.out.println("Propietario: " + propietario);
        System.out.println("Fecha cumpleaños: " + fechaCumpleaños);
        System.out.println("Ultima vacuna: " + fechaUltimaVacuna);
    }
    
}
